include/aie/Dialect/AIE/IR/AIE.td

```diff
@@ -1056,14 +1059,17 @@ def AIE_UseLockOp: AIE_Op<"useLock", []> {
   }];
   let hasVerifier = 1;
   let builders = [
-    OpBuilder<(ins "::mlir::Value":$lock, "uint32_t":$value, "xilinx::AIE::LockAction":$action),
+    OpBuilder<(ins "::mlir::Value":$lock,
+                   "uint32_t":$value,
+                   "xilinx::AIE::LockAction":$action),
     [{
       build($_builder, $_state, lock, value, action, nullptr);
     }]>
   ];
 
   let extraClassDeclaration = [{
-    bool acquire() { return (getAction() == LockAction::Acquire); }
+    bool acquire_eq() { return (getAction() == LockAction::Acquire); }
```

Comment:
seems like if this is going to change, it should change everywhere?  Otherwise, seems like this should remain consistent with the IR?